options/locale/locale_en-US.ini

```diff
@@ -1434,6 +1434,7 @@ issues.choose.ignore_invalid_templates = Invalid templates have been ignored
 issues.choose.invalid_templates = %v invalid template(s) found
 issues.choose.invalid_config = The issue config contains errors:
 issues.no_ref = No Branch/Tag Specified
+issues.no_results = No results found.
```

Comment:
Why make a new duplicate translation string?

```
issues.no_results = No results found.
pulls.no_results = No results found.
```

This will lead to unnecessary work for the translators. Just keep the old key, even if it's not ideal.